### PR TITLE
change link to new label for up-for-grabs

### PR DESCRIPTION
**Pull request by borsna**
see dotnet/corefx#17533

### CODE REVIEW

**Comment by zhenlan** [comment]
@StephenBonikowsky, is our CI build of "OuterLoop Ubuntu14.04 Debug" broken somehow? Not related to this PR.

**Comment by StephenBonikowsky** [comment]
Not from our part (that I've been able to figure out), they seem to fail regularly and will eventually pass on re-running. I can follow-up with infrastructure team to see if this is common with other repos as well or just us.

**Comment by zhenlan** [comment]
Thanks @StephenBonikowsky. Yes, please follow up so we make sure we don't miss something.

File: Documentation/issue-guide.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ We use GitHub labels on our issues in order to classify them.  We have the follo
  * [test bug](https://github.com/dotnet/wcf/labels/test%20bug) is a label we assign to all issues related to testing. We label these further with "infrastructure" (for test infrastructure changes) or "enhancement" (for missing tests or those that need enhancement).
 
 **Ownership**: These labels are used to specify who owns specific issue. Issues without an ownership tag are still considered "up for discussion" and haven't been approved yet. We have the following different types of ownership:
- * [up for grabs](https://github.com/dotnet/wcf/labels/up%20for%20grabs): Small sections of work which we believe are well scoped. These sorts of issues are a good place to start if you are new.  Anyone is free to work on these issues.
+ * [up for grabs](https://github.com/dotnet/wcf/labels/up-for-grabs): Small sections of work which we believe are well scoped. These sorts of issues are a good place to start if you are new.  Anyone is free to work on these issues.
```
Comment (karelz) [inline Documentation/issue-guide.md]: Change the visible name also to "up-for-grabs" (replace spaces with -) to provide unified clarity.

File: Documentation/issue-guide.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ We use GitHub labels on our issues in order to classify them.  We have the follo
...
  * [feature approved](https://github.com/dotnet/wcf/labels/feature%20approved): Larger scale issues.  Like up for grabs, anyone is free to work on these issues, but they may be trickier or require more work.
```
Comment (karelz) [inline Documentation/issue-guide.md]: change "up for grabs" to up-for-grabs

**Comment by karelz** [comment]
I missed it's merged already. I made the changes here: https://github.com/dotnet/wcf/commit/1f343b2d2b3d3f17221215d936eb86381482a9d8

**Pull request merged**